### PR TITLE
Symfony: HTTP client tracing

### PR DESCRIPTION
**Pull request by cleptric**

### CODE REVIEW

File: src/platforms/php/guides/symfony/performance/instrumentation/automatic-instrumentation.mdx [hunk 1]
```diff
@@ -36,7 +38,7 @@ It is enabled by default. To disable tracking database queries, set `enabled` to
 sentry:
   tracing:
     dbal: # DB queries
-      enabled: true
+      enabled: false
 ```
 
 ### Twig
```
Comment (imatwawana) [inline src/platforms/php/guides/symfony/performance/instrumentation/automatic-instrumentation.mdx:44]:
```suggestion
## Twig
```

File: src/platforms/php/guides/symfony/performance/instrumentation/automatic-instrumentation.mdx [hunk 2]
```diff
@@ -49,7 +51,7 @@ It is enabled by default. To disable it, set `enabled` to `false`.
 sentry:
   tracing:
     twig: # templating engine
-      enabled: true
+      enabled: false
 ```
 
 ### Cache
```
Comment (imatwawana) [inline src/platforms/php/guides/symfony/performance/instrumentation/automatic-instrumentation.mdx:57]:
```suggestion
## Cache
```

File: src/platforms/php/guides/symfony/performance/instrumentation/automatic-instrumentation.mdx [hunk 3]
```diff
@@ -62,5 +64,36 @@ It is enabled by default. To disable it, set `enabled` to `false`.
...
+It is enabled by default. To disable it, set `enabled` to `false`.
+
+```yaml {filename:config/packages/sentry.yaml}
+sentry:
+  tracing:
+    http_client: # Symfony HTTP client
+      enabled: false
+```
+
+As the HTTP client tracing is based on Symfony's [TraceableHttpClient](https://github.com/symfony/symfony/blob/558fec00b0fa92dc1499434355e5a35de162bbe7/src/Symfony/Component/HttpClient/TraceableHttpClient.php) there are a few things to consider:
```
Comment (imatwawana) [inline src/platforms/php/guides/symfony/performance/instrumentation/automatic-instrumentation.mdx:83]:
```suggestion
As the HTTP client tracing is based on Symfony's [TraceableHttpClient](https://github.com/symfony/symfony/blob/558fec00b0fa92dc1499434355e5a35de162bbe7/src/Symfony/Component/HttpClient/TraceableHttpClient.php), you must consider the following:
```

File: src/platforms/php/guides/symfony/performance/instrumentation/automatic-instrumentation.mdx [hunk 4]
```diff
@@ -62,5 +64,36 @@ It is enabled by default. To disable it, set `enabled` to `false`.
...
+
+- To get correct span timings, do not assign the response of the request to a variable if you do not intend to use it.
```
Comment (imatwawana) [inline src/platforms/php/guides/symfony/performance/instrumentation/automatic-instrumentation.mdx:85]:
```suggestion
- To get correct span timings, don't assign the response of the request to a variable if you don't intend to use it.
```

**Review by imatwawana — COMMENTED** [review]
I made some minor wording tweaks, but have a structural question. Also, please update the heading level on line 31 from 3 to 2 (remove one of the hashes). We should avoid skipping heading levels.

**Comment by cleptric** [comment]
@imatwawana Thank you 🙂

**Review by imatwawana — APPROVED** [review]
LGTM!

**Pull request merged**